.github/workflows/test.yml

```
@@ -32,12 +32,10 @@ jobs:
       with:
         go-version-file: go.mod
     - name: Install gotestfmt
```

I think the `name` should be also update to reflect the change
```suggestion
    - name: Install tparse
```